frontend/public/components/edit-yaml.jsx

```diff
@@ -299,7 +299,11 @@ export const EditYAML = connect(stateToProps)(
         action(model, obj, newNamespace, newName)
           .then(o => {
             if (redirect) {
-              history.push(this.props.redirectURL || resourceObjPath(o, referenceFor(o)));
+              const redirectURL = this.props.redirectURL;
+              history.push(
+                (redirectURL && _.isFunction(redirectURL) ? redirectURL(o) : redirectURL) ||
```

Comment:
New prop should be introduced here, the `redirectURL` should stay string-only.

Please see comments in #1820.

Reply:
:+1:

Reply:
@mareklibra `resourceObjPath` prop, WDYT ?